database/raft/raft.go

```diff
@@ -175,6 +175,7 @@ func Start(laddr, dir, bootURL string, httpClient *http.Client, useTLS bool) (*S
 
 	// TODO(kr): grpc
 	sv.mux.HandleFunc("/raft/join", sv.serveJoin)
+	sv.mux.HandleFunc("/raft/add-allowed-member", sv.serveAddAllowedMember)
```

Comment:
Is this gonna need to move out of this package so it can also edit the grants list?